### DISCUSSION TITLE
Does dropbar.nvim supports json files

### DISCUSSION THREAD

**Discussion by harshv5094** [Bekaboo/dropbar.nvim#6] [General]
Hey, I wanted to ask some questions related to dropbar.nvim. Does your project support JSON files? I want to use this plugin as I'm daily editing my 'Oh-My-Posh' File.
![image](https://github.com/Bekaboo/dropbar.nvim/assets/51878195/da194e4c-7749-4c4c-aeac-6ded5477aed0)

**Comment by harshv5094** [comment]
By the way, This screenshot is from when I Installed the plugin for the first time.

**Comment by Bekaboo** [comment]
`dropbar.nvim` requires nvim ≥ 0.10.0, see https://github.com/Bekaboo/dropbar.nvim#requirements
